src/Integration/Helpers/IDEWindowService.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.ComponentModel.Composition;
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.Integration.Helpers
+{
+    [Export(typeof(IIDEWindowService))]
+    internal class IDEWindowService : IIDEWindowService
```

Comment:
I considered merging this service with `IToolWindowService` which also only has one method; both methods could be logically grouped into a "window services" component. I think I prefer keeping them separate, but I don't have a strong feeling either way.

Reply:
No I agree, they serve a different functionality, I'd keep them separated